### PR TITLE
974 color issue

### PR DESCRIPTION
**Pull request by Mussuranae**
Resolve color issue #974 and issue #881 for mail notification.
Review of the light theme

### CODE REVIEW

File: libs/movie/src/lib/movie/form/root/root.component.scss [hunk 1]
```diff
@@ -36,48 +29,32 @@ mat-card.movie-form-container {
...
-      box-shadow: none;
-      font-weight: 700;
-      font-size: 12px;
-      width: 200px;
-      height: 45px;
-    }
+.movie-card-actions {
+  position: fixed;
+  bottom: 0px;
+  margin: 0px !important;
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/form/root/root.component.scss]: Try to remove it

File: libs/movie/src/lib/movie/form/root/root.component.scss [hunk 2]
```diff
@@ -36,48 +29,32 @@ mat-card.movie-form-container {
...
+  z-index: 2;
+  background: white;
+  width: 100%;
+  padding: 8px !important;
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/form/root/root.component.scss]: same

File: libs/movie/src/lib/movie/form/root/root.component.scss [hunk 3]
```diff
@@ -36,48 +29,32 @@ mat-card.movie-form-container {
...
-        margin-bottom: 3px;
-      }
-    }
-  }
+.mat-tab-header {
+  position: fixed;
+  z-index: 2;
+  background: white;
+  width: 100%;
+  margin-top: -2px;
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/form/root/root.component.scss]: Delete it

File: libs/ethers/src/lib/wallet/pages/wallet-no-key/wallet-no-key.component.html [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <section fxLayout="row" fxLayoutAlign="center center">
   <h2>You haven't any keys</h2>
-  <a mat-flat-button routerLink="/layout/o/account/wallet/import">
+  <a mat-stroked-button color="primary" routerLink="/layout/o/account/wallet/import">
     <mat-icon class="material-icons-outlined">cloud_upload</mat-icon>
     Import Key
   </a>
-  <a mat-flat-button routerLink="/layout/o/account/wallet/add">
+  <a mat-stroked-button color="primary" routerLink="/layout/o/account/wallet/add">
```
[the remark this answers is not in the document]
Comment (Mussuranae) [reply libs/ethers/src/lib/wallet/pages/wallet-no-key/wallet-no-key.component.html]: Color is needed, my bad.

File: apps/catalog/marketplace/marketplace/src/app/distribution-right/create/create.component.html [hunk 5]
```diff
@@ -1,23 +1,24 @@
...
+        <span>a film by {{ director.firstName }} {{ director.lastName }} <span *ngIf="!isLast">&</span>
+        </span>
+      </div>
     </div>
   </header>
-  <article fxLayout="row">
-    <aside>
+
+  <article fxLayout="row" class="page-content">
+    <div>
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/distribution-right/create/create.component.html]: You are mixing ``div`` with ``section`` on the same level in an ``article``.

File: apps/catalog/marketplace/marketplace/src/app/distribution-right/create/create.component.scss [hunk 6]
```diff
@@ -4,10 +4,10 @@
   flex-direction: row;
 }
 
-aside {
-  min-width: 450px;
-  width: 25%;
-  padding: 10px;
+.page-content {
+  margin-left: 200px;
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/distribution-right/create/create.component.scss]: You add margin right-left 200px in the main article, I think it's broken in resposive

File: libs/auth/src/lib/components/email-verify/email-verify.component.html [hunk 7]
```diff
@@ -1,12 +1,11 @@
-<!-- TODO: issue#881 uncomment after Toronto -->
```
Comment (Hugo-Hontans) [inline libs/auth/src/lib/components/email-verify/email-verify.component.html]: Just think to close this issue if you resolve it

File: libs/organization/src/lib/pages/organization-activity-view/organization-activity-view.component.html [hunk 8]
```diff
@@ -1,9 +1,9 @@
 <editable-sidenav (closed)="opened = false" [opened]="opened">
     <ng-container content>
-      <h2 fxLayout="row" fxLayoutAlign="start center">
-        <span fxFlex="noshrink">Activity Report</span>
+      <section fxLayout="row" fxLayoutAlign="start center">
```
Comment (Hugo-Hontans) [inline libs/organization/src/lib/pages/organization-activity-view/organization-activity-view.component.html]: Use a header tag if you just have a title inside

File: libs/organization/src/lib/pages/organization-admin-view/organization-admin-view.component.html [hunk 9]
```diff
@@ -4,10 +4,10 @@
     <ng-container content>
 
       <!-- Admin Tab -->
-      <h2 fxLayout="row" fxLayoutAlign="start center">
-        <span fxFlex="noshrink">{{organizationName}}</span>
+      <section fxLayout="row" fxLayoutAlign="start center">
```
Comment (Hugo-Hontans) [inline libs/organization/src/lib/pages/organization-admin-view/organization-admin-view.component.html]: same

**Pull request merged**